### PR TITLE
docs(CONTRIBUTING): add section on how to create schemas

### LINKED ISSUE #643: Document how we get from a new webhook event to a new schema and to new types
Follow up to https://github.com/octokit/webhooks/pull/641#discussion_r898325831

### PR DESCRIPTION
**Pull request by wolfy1339**
These are basic steps to creating a schema.
Fixes #643

### CODE REVIEW

**Comment by wolfy1339** [comment]
Blocked by #641, as the current cache has multiple entries for the new `security_and_analysis` event, and the `index.json` file didn't get generated in #634 which causes the tests to fail.

Would need to override the required tests for merge.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -39,6 +39,52 @@ When you send a pull request, make sure that the `index.json` file is up-to-date
 with the changes in the `payload-examples/api.github.com/` folder, otherwise the
 tests will fail.
 
+## Creating a new schema
+
+The webhook schemas are [JSON schemas](https://json-schema.org/) that are manually created using example payloads for webhook events.
+
+- Create a new folder for the event, if it doesn't already exist
+- Create a new JSON file named `<action>.schema.json` if the event has an `action` property, otherwise name it `event.schema.json`
+- The `$id` property's value should be in the format `<eventName>$action`, where `<eventName>` is the name of the event, and `action` is the value of the `action` property of the webhook payload or simply `event` if there is no `action` property for the event
```
Comment (G-Rath) [inline CONTRIBUTING.md:48]: For consistency I think we should use `<action>` here:

```suggestion
- The `$id` property's value should be in the format `<eventName>$<action>`, where `<eventName>` is the name of the event, and `<action>` is the value of the `action` property of the webhook payload or simply `event` if there is no `action` property for the event
```

nit: though I'd probably suggest rephrasing this to be something like:

> The `$id` property should start with the event name, followed by the `action` name separated with `$` if the event has an `action` property, e.g. `pull_request$created`

**Review by G-Rath — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (wolfy1339) [reply CONTRIBUTING.md]: I went with your rephrased version

**Comment by wolfy1339** [comment]
I believe this is ready to merge. (The test failures are unrelated to this PR)

**Review by gr2m — APPROVED** [review]
This is really great, thank you  so much for putting it together!

**Comment by oscard0m** [comment]
Arrived late to the review but thanks for this @wolfy1339 ! ❤️ 👏🏽

**Pull request merged**